docs/GlutenUsage.md

```diff
@@ -12,11 +12,10 @@ Based on the different environment, there are some parameters can be set via -D
 | build_velox_from_source                               | Enable or Disable building Velox from a specific velox github repository. A default installed path will be in velox_home                                                           | OFF                                                |
 | backends-velox                                        | Add -Pbackends-velox in maven command to compile the JVM part of Velox backend                                                                                                     | false                                              |
 | backends-clickhouse                                   | Add -Pbackends-clickhouse in maven command to compile the JVM part of ClickHouse backend                                                                                           | false                                              |
-| build_velox                                           | Enable or Disable building the CPP part of Velox backend                                                                                                                           | OFF                                                |
-| velox_home (only valid when build_velox is ON)        | The path to the compiled Velox project. When building Gluten with Velox, if you have an existing Velox, please set it.                                                             | /PATH_TO_GLUTEN/ep/build-velox/build/velox_ep      |
+| build_velox_backend                                           | Enable or Disable building the CPP part of Velox backend                                                                                                                           | OFF                                                |
+| velox_home (only valid when build_velox_backend is ON)        | The path to the compiled Velox project. When building Gluten with Velox, if you have an existing Velox, please set it.                                                             | /PATH_TO_GLUTEN/ep/build-velox/build/velox_ep     |
 | compile_velox(only valid when velox_home is assigned) | recompile exising Velox use custom compile parameters                                                                                                                              | OFF                                                |
 | velox_build_type                                      | The build type Velox was built with from source code. Gluten uses this value to locate the binary path of Velox's binary libraries.                                                | release                                            |
```

Comment:
Just a tiny suggestion. Can you leave some comment to tell user all available velox build types (release, debug, relWithDebInfo)? Thanks!

Reply:
fixed it. 
add (option: release/debug/relWithDebInfo)

Reply:
is this issue https://github.com/oap-project/gluten/issues/516 addressed by this PR? Ideally I use something like 

mvn .... -Dtarget_build=release/debug/relWithDebInfo, then all the libraries we build like arrow, velox, protobuf, etc. all have related build type.

Reply:
yes, this pr will address #516  and #517.